relayer/relays/beefy/polkadot-listener.go

```diff
@@ -161,17 +122,3 @@ func (li *PolkadotListener) queryBeefyAuthorities(blockHash types.Hash) ([]subst
 
 	return authorities, nil
 }
-
-func (li *PolkadotListener) queryBeefyNextAuthoritySet(blockHash types.Hash) (types.BeefyNextAuthoritySet, error) {
-	var nextAuthoritySet types.BeefyNextAuthoritySet
-	storageKey, err := types.CreateStorageKey(li.conn.Metadata(), "MmrLeaf", "BeefyNextAuthorities", nil, nil)
-	ok, err := li.conn.API().RPC.State.GetStorage(storageKey, &nextAuthoritySet, blockHash)
-	if err != nil {
-		return nextAuthoritySet, err
-	}
-	if !ok {
-		return nextAuthoritySet, fmt.Errorf("beefy nextAuthoritySet not found")
-	}
-
-	return nextAuthoritySet, nil
-}
```

Comment:
I need this function for https://github.com/Snowfork/snowbridge/pull/1215, but no need to revert here I can address it in that PR.